CMakeLists.txt

```diff
@@ -130,6 +137,10 @@ set(REALSENSE_CPP
     src/media/playback/playback_sensor.cpp
     )
 
+if(BUILD_EASYLOGGINGPP)
```

Comment:
I would combine the two parts into one so that all EL++ related part is in one place - easier to maintain and follow.

Reply:
ok.